homeassistant/components/sensor/fritzbox.py

```diff
@@ -0,0 +1,77 @@
+"""
+Support for AVM Fritz!Box smarthome temperature sensor only devices.
+
+For more details about this component, please refer to the documentation at
+http://home-assistant.io/components/sensor.fritzbox/
+"""
+import logging
+
+import requests
+
+from homeassistant.components.fritzbox import DOMAIN as FRITZBOX_DOMAIN
+from homeassistant.components.fritzbox import (
+    ATTR_STATE_DEVICE_LOCKED, ATTR_STATE_LOCKED)
+from homeassistant.helpers.entity import Entity
+from homeassistant.const import TEMP_CELSIUS
+
+DEPENDENCIES = ['fritzbox']
+
+_LOGGER = logging.getLogger(__name__)
+
+
+def setup_platform(hass, config, add_entities, discovery_info=None):
+    """Set up the Fritzbox smarthome sensor platform."""
+    _LOGGER.debug("Initializing fritzbox temperature sensors")
+    devices = []
+    fritz_list = hass.data[FRITZBOX_DOMAIN]
+
+    for fritz in fritz_list:
+        device_list = fritz.get_devices()
+        for device in device_list:
+            if (device.has_temperature_sensor
+                    and not device.has_switch
+                    and not device.has_thermostat):
+                devices.append(FritzBoxTempSensor(device, fritz))
+
+    add_entities(devices)
+
+
+class FritzBoxTempSensor(Entity):
+    """The entity class for Fritzbox temperature sensors."""
+
+    def __init__(self, device, fritz):
+        """Initialize the switch."""
+        self._device = device
+        self._fritz = fritz
+
+    @property
+    def name(self):
+        """Return the name of the device."""
+        return self._device.name
```

Comment:
Is this doing I/O? If not (assuming update() populates them), this can be merged as it is. Otherwise this, `state` and `device_state_attributes` need be modified not to do any I/O.

Reply:
Great, merged! :1st_place_medal: Please update the `ha_release` part in the docs and let's get that merged in, too.